product_docs/docs/pem/9/managing_certificates_copy.mdx

```diff
@@ -0,0 +1,436 @@
+
+    ```shell
+    /usr/pgsql-x/bin/psql -U postgres -d pem --no-psqlrc -t -A -c "SELECT public.openssl_rsa_generate_key(1024)" >> /var/lib/pgsql/x/data/server.key
+    ```
+
+    After creating the `server.key` file, `cat` the contents to the variable `SSL_KEY` for use when generating the `server.crt` file and modify the privileges on the `server.key` file:
+
+    ```shell
+    SSL_KEY=$(cat /var/lib/pgsql/x/data/server.key)
+
+    chmod 600 /var/lib/pgsql/x/data/server.key
+    ```
+
+8.  Use the `SSL_KEY` to generate the server certificate. Save the certificate in the `server.crt` file. For simplicity, first place the SQL query into a temporary file with a unique name:
+
+    ```shell
+    echo "SELECT openssl_csr_to_crt(openssl_rsa_key_to_csr('${SSL_KEY}',
+    'PEM','US', 'MA', 'Bedford', 'Postgres Enterprise Manager',
+    'support@enterprisedb.com'),
+    '/var/lib/pgsql/x/data/ca_certificate.crt',
+    '/var/lib/pgsql/x/data/ca_key.key')" > /tmp/_random.$$
+
+     /usr/pgsql-x/bin/psql -U postgres -d pem --no-psqlrc -t -A -f /tmp/_random.$$ >> /var/lib/pgsql/x/data/server.crt
+    ```
+
+9.  Modify the privileges on the `server.crt` file, and delete the temporary file:
+
+    ```shell
+    chmod 600 /var/lib/pgsql/x/data/server.crt
+
+    rm -f /tmp/_random.$$
+    ```
+
+10. Restart the Postgres server:
+
+       On Linux with `init.d`, for example, on a Centos6 host:
+
+    ```shell
+    /etc/init.d/postgresql-x restart
+    ```
+
+    On Linux with `systemd`, for example, on a Centos7 host:
+
+    ```shell
+    systemctl restart postgresql-x
+    ```
+
+## Replacing PEM self-signed certificates and keys with trusted CA certificates and keys
+
+Before replacing the PEM self-signed certificates and keys with the trusted third-party CA certificates and keys, you must get those certificates using the steps indicated in the third-party CA. 
+
+After obtaining the trusted third-party CA certificates and keys, perform the following steps: 
+
+- Replace PEM self-signed certificates and keys with trusted CA-signed certificates and keys
+- Install trusted CA root and intermediate certificates
+- Install trusted CA CRL
+- Disable CRL usage by the server if the trusted third-party CA does not provide CRL
+
+### Example: Replacing the server and agent certificates
+
+This example shows you how to replace the self-signed certificates with the trusted CA certificates.
+
+We have PEM 9.1 server on CentOS 7 with EPAS 14 as the backend server and a standalone agent installed on another machine. We have to replace the certificates for the backend server, standalone agent and default agent.
+
+Perform the steps to update the self-signed SSL certificates:
+
+1.  Make sure both the PEM agents and the PEM server are up and running.
+
+1.  Take a backup of the existing SSL keys and certificates. The SSL keys and certificates are stored in the `data` directory under your PEM installation. For example, the default location on a Linux system is:
```

Comment:
Since we are presenting this section as an example, don't use "for example" just provide the step you would use for EPAS 14 on CentOS.

Either removed the default location content or say "Since we are using CentOS in this example, the default location is ..."